specification/compute/resource-manager/Microsoft.Compute/DiagnosticRP/preview/2025-02-01-preview/diagnostic.json

```diff
@@ -979,7 +979,29 @@
           ],
           "x-ms-enum": {
             "name": "RestrictionsFilterType",
-            "modelAsString": false
+            "modelAsString": false,
```

Comment:
Are any SDKs generated from this? If they are or plan to be, I advise changing modelAsString to true as that will allow you to add more values to this enum without introducing breaking changes to the sdks.

Reply:
AS of now no SDK will be generated and as we plan on SDK it will be newer version. So at that point will make this change.